pymc3/tests/test_distributions.py

```diff
@@ -206,6 +207,7 @@ def product(domains, n_samples=-1):
 
 Bool = Domain([0, 0, 1, 1], "int64")
 
+Scale = Domain([0.5,1,1.5])
```

Comment:
Do you need a new Domain type? Can't the `Rplus` be used? Also the edge values should reflect the actual parameter boundaries. Is 0.5 and 1.5, the minimum and maximum values that the `scale` parameter can take?

Reply:
> Also the edge values should reflect the actual parameter boundaries. Is 0.5 and 1.5, the minimum and maximum values that the `scale` parameter can take?

The code that we are using for asymmetric Laplace has three input parameters while the one that is taken from scipy.stats.laplace_asymmetric has only two parameters. so there are two possibilities . either we have to fix the scale to be 1 or we will have to change the referenced code. So, I thought of fixing the scale to be 1. as far as the minimum and maximum value of scale is concerned, it can be anything greater than zero. Anyway, I will change that.
we can only use `Rplus` if we modify the referenced logpdf code.
so, what should I do?

Reply:
In that case I think you can create a specific Domain inside your test function (as this is unlikely to be useful for other distributions).

```python
    def test_laplace_asymmetric(self):
        self.pymc3_matches_scipy(
            AsymmetricLaplace,
            R,
            {"scale": Domain([0, 1, inf]), "symmetry": Rplus},
            laplace_asymmetric_logpdf,
        )
```

Reply:
okay,  understood
Thank you